package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Privly",
+  "name": "privly",
+  "version": "0.5.0",
+  "description": "Privly enables you to post and read private content through any website. Only the people you designate can read your content, and the host page never has access. This extension is still experimental. For more information on Privly development, go to http://www.privly.org",
```

Comment:
"Privly enables you to post and read private content through any website without those websites storing or reading your content. This extension is still experimental. For more information on Privly development, go to https://priv.ly/pages/develop"